ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -128,7 +128,7 @@ export class RemoteEnvironmentService extends EnvironmentService {
         this.log.debug(`initializing ${executor.name}`);
 
         // Create root working directory after executor is ready
-        const nniRootDir: string = executor.joinPath(executor.getTempPath(), 'nni');
+        const nniRootDir: string = executor.joinPath(executor.getTempPath(), 'nni-experiments');
```

Comment:
@SparkSnail please check this

Reply:
yes, this should be chaned to `nni-experiments`